contracts/tools/testing/slither/Slither.sol

```diff
@@ -13,4 +13,6 @@ import {IexecPoco2Delegate} from "../../../modules/delegates/IexecPoco2Delegate.
  * @dev This contract aggregates multiple delegate contracts into a single entry point
  *      to facilitate static analysis using Slither.
  */
+//slither-disable-start unused-state
```

Comment:
Not sure to understand why we need that annotation :thinking:

Reply:
` - [unused-state](#unused-state) (25 results) (Informational)`
`ERC1538Store.m_funcs (contracts/Store.v8.sol#38) is never used in Slither (contracts/tools/testing/slither/Slither.sol#17)`

Reply:
It just say that a lot of variable are not used in this specifc contract `Slither.sol` wich is normal 
So I prefer skip the slither check to have less verbosity in the report